lib/rubocop/sketchup/cop/suggestions/command_title.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module SketchupSuggestions
+      # SketchUp command titles should be in title case, e.g. "Make Component"
+      # and "Follow Me".
+      #
+      # In English, capitalize the first letter of each word. Other languages
+      # may have different rules.
+      class CommandTitle < SketchUp::Cop
+
+        def_node_matcher :init_entity?, <<-PATTERN
+          (send (const (const nil? :UI) :Command ) :new ... )
+        PATTERN
+
+        MESSAGE = 'Use title case in command titles. '\
+                  'In English, capitalize the first letter of each word.'
+
+        def on_send(node)
+          return unless init_entity?(node)
+          return if node.arguments.first.str_content.nil?
```

Comment:
Is there a more readable way to see of an argument is a string token?

I want to catch `method("some_string")` but ignore `method(some_variable)`.

This line gets the job done but is not very clear.

Reply:
Sounds like all string literal nodes will be `RuboCop::AST::StrNode`
https://rubydoc.info/github/rubocop/rubocop-ast/RuboCop/AST/StrNode

This appear to also include heredoc strings.

Reply:
That being said, you are using a node matcher only to query if the node is something - you can use it to also capture.

First I think we can replace `...` with something that captures string literals:  `(str $_)`  (I'm not 100% sure about the syntax). This might also not capture all string literals, I'd have to dig deeper.

But notice we added `(str` into the pattern, making the pattern check the type instead of the outer logic. Then we have `$` - it captures the node. You can see an example of this in SketchUpBugs/RenderMode. Also in SketchupRequirements/MinimalRegistration.

Reply:
There is also `node.str_type?` (SketchupSuggestions/FileEncoding)